src/components/IOUConfirmationList.js

```diff
@@ -258,6 +321,28 @@ class IOUConfirmationList extends Component {
         ];
     }
 
+    /**
+     * Adds Venmo, if available, as the second option in the menu of payment options
+     */
+    addVenmoPaymentOptionToMenu() {
+        // Add Venmo option
+        if (this.props.localCurrencyCode === CONST.CURRENCY.USD && this.state.participants[0].phoneNumber.length > 0 && isValidUSPhone(this.state.participants[0].phoneNumber)) {
```

Comment:
This line failed on web. I assume because I had no phone number?
![2021-09-28_16-28-05](https://user-images.githubusercontent.com/42391420/135174288-681bea47-ad5f-45fc-993f-752c98d6c61a.png)

Reply:
It caused the app to crash btw